torch/csrc/lazy/core/helpers.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
```

Comment:
Sounds like we could consolidate the file and util.h. Can you leave a TODO as a follow up?

Reply:
Agree. I have mentioned it in the PR description, but let me add a TODO to make it explicit.